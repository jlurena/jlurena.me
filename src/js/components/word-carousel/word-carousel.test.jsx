/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'
import WordCarousel from '.'

test('<WordCarousel/> Rendering', () => {
  const words = ['Hello', 'World', 'Goodbye']

  const component = renderer.create(
    <WordCarousel words={words} />
  )

  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
