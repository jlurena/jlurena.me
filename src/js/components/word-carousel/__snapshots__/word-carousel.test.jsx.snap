// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WordCarousel/> Rendering 1`] = `
[
  <span
    className="word"
  >
    Hello
  </span>,
  <span
    className="word"
  >
    World
  </span>,
  <span
    className="word"
  >
    Goodbye
  </span>,
]
`;
